api_tests/tests/dry/sanity.ts

```diff
@@ -0,0 +1,167 @@
+                    beta_ledger_redeem_identity: "",
+                    alpha_expiry: 123456789,
+                    beta_expiry: 123456789,
+                    peer: "QmPRNaiDUcJmnuJWUyoADoqvFotwaMRFKV2RyZ7ZVr1fqd",
+                });
+
+            expect(res).to.have.status(400);
+            expect(res).to.have.header(
+                "content-type",
+                "application/problem+json"
+            );
+            expect(res.body.title).to.equal("Invalid body.");
+        });
+    });
+    it("returns-invalid-body-for-bad-json", async function() {
+        await oneActorTest(async function({ alice }) {
+            const res = await request(alice.cndHttpApiUrl())
+                .post("/swaps/rfc003")
+                .send({
+                    garbage: true,
+                });
+
+            expect(res).to.have.status(400);
+            expect(res).to.have.header(
+                "content-type",
+                "application/problem+json"
+            );
+            expect(res.body.title).to.equal("Invalid body.");
+        });
+    });
+    it("alice-has-empty-peer-list", async function() {
+        await oneActorTest(async function({ alice }) {
+            const res = await request(alice.cndHttpApiUrl()).get("/peers");
+
+            expect(res).to.have.status(200);
+            expect(res.body.peers).to.have.length(0);
+        });
+    });
+    it("returns-listen-addresses-on-root-document", async function() {
+        await oneActorTest(async function({ alice }) {
+            const res = await request(alice.cndHttpApiUrl()).get("/");
+
+            expect(res.body.id).to.be.a("string");
+            expect(res.body.listen_addresses).to.be.an("array");
+            // At least 2 ipv4 addresses, lookup and external interface
+            expect(res.body.listen_addresses.length).to.be.greaterThan(1);
+        });
+    });
+    it("can-fetch-root-document-as-siren", async function() {
+        await oneActorTest(async function({ alice }) {
+            const res = await request(alice.cndHttpApiUrl()).get("/");
+
+            expect(res).to.have.status(200);
+            expect(res.body).to.be.jsonSchema(sirenJsonSchema);
+        });
+    });
+    it("returns-listen-addresses-on-root-document-as-siren", async function() {
+        await oneActorTest(async function({ alice }) {
+            const res = await request(alice.cndHttpApiUrl())
+                .get("/")
+                .set("accept", "application/vnd.siren+json");
+
+            expect(res.body.properties.id).to.be.a("string");
+            expect(res.body.properties.listen_addresses).to.be.an("array");
+            // At least 2 ipv4 addresses, lookup and external interface
+            expect(
+                res.body.properties.listen_addresses.length
+            ).to.be.greaterThan(1);
+        });
+    });
+    it("returns-links-to-create-swap-endpoints-on-root-document-as-siren", async function() {
```

Comment:
Would be nice if we could just specify regular sentences here and we just replace the dashes with hyphens for the log file.

Reply:
Replace spaces and slashes with dashes*

Reply:
That'd be a lot of work. I prefer not doing it in this PR 😬